projects/addon-commerce/components/input-card-grouped/input-card-grouped.providers.ts

```diff
@@ -0,0 +1,55 @@
+import {inject, InjectionToken} from '@angular/core';
+import {WINDOW} from '@ng-web-apis/common';
+import {
+    TUI_CARD_EXPIRY_TEXTS,
+    TUI_CARD_NUMBER_TEXTS,
+} from '@taiga-ui/addon-commerce/tokens';
+import {typedFromEvent} from '@taiga-ui/cdk';
+import {Observable, of} from 'rxjs';
+import {map, startWith, withLatestFrom} from 'rxjs/operators';
+import {MEDIA} from '../../../core';
+
+export interface TuiCardGroupedTexts {
+    readonly cardNumberText: string;
+    readonly expiryText: string;
+    readonly cvcText: string;
+}
+
+export const TUI_CARD_CVC = new InjectionToken<Observable<[string, string]>>(
```

Comment:
This one wasn't there before? If so it should probably go into same place as others

Reply:
This was hardcoded in the template. The other tokens are in tokens/i18n.ts. Do you think it is worth moving it to i18n.ts?

Reply:
Nah, just move to `@taiga-ui/addon-commerce/tokens`, I think the default text works for any language.